ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx

```diff
@@ -101,7 +101,7 @@ const getDisplayValues = ({
   } else if (isSmartTransactionCancelled) {
     return {
       title: t('smartTransactionCancelled'),
-      description: t('smartTransactionCancelledDescription', [countdown]),
```

Comment:
Perfect example! This translation doesn't need an argument!

Thankfully from the files I could test (only typescript files), there weren't any issues where we were using translations but forgetting to pass args.